### PR TITLE
Add to home / Feature/spa 1094

### PR DESCRIPTION
**Pull request by znikola**

### CODE REVIEW

File: projects/storefrontlib/src/lib/ui/layout/header/mobile-menu/mobile-menu.component.ts [hunk 1]
```diff
@@ -25,6 +27,17 @@ export class MobileMenuComponent implements OnDestroy {
     }
   }
 
+  ngOnInit() {
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/layout/header/mobile-menu/mobile-menu.component.ts]: Implement life cycle hook interface OnInit for method ngOnInit

File: projects/storefrontlib/src/lib/ui/layout/header/mobile-menu/mobile-menu.component.html [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+      <div class="y-mobile-menu__locale">
+        <y-language-selector></y-language-selector>
+        <y-currency-selector></y-currency-selector>
       </div>
-
-      <div *ngIf="showMenu" class="y-mobile-menu__overlay" (click)="toggleMenu()"></div>
-</div>
\ No newline at end of file
+      <!-- temporary solution -->
+      <div *ngIf="canAddToHomeScreen">
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/layout/header/mobile-menu/mobile-menu.component.html]: I could only get this to display by removing the if. Once I can see it, it does nothing

File: projects/storefrontlib/src/lib/ui/layout/main/main.component.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/layout/main/main.component.ts]: What is this component for?

**Comment by filrak** [comment]
Hey @LTiger14 this is old evrsion of this feature. Can you please review https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/50 instead? (I know about merge conflicts, will be solved on Monday).

Also it looks like travis is installing older version of ng-bootstrap and build is not passing ;/

**Comment by filrak** [comment]
Also can you PM me so I can explain you how ATHS feature works? It;'s a bit tricky

**Comment by znikola** [comment]
Closing, as this was an old branch

**Pull request closed without merging**